release-tasks.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
I've been seeing more things moving towards `/usr/bin/env bash`. Probably doesn't matter in this case, but that [respects any local overrides](https://stackoverflow.com/a/2429517/372479) in `PATH`.

```suggestion
#!/usr/bin/env bash
```